site/tutorials/gettings_started.md

```diff
@@ -131,13 +131,13 @@ This time we set the field type to be `quantitative` because the values in field
   },
   "mark": "point",
   "encoding": {
-    "x": {"field": "a", "type": "ordinal"},
+    "x": {"field": "a", "type": "nominal"},
     "y": {"field": "b", "type": "quantitative"}
   }
 }
 </div>
 
-Now we can see the raw data points. Note that Vega-Lite automatically adds grid lines to the y-axis to help you compare the magnitude of the `b` values.
+Now we can see the raw data points. Note that Vega-Lite automatically adds grid lines to the y-axis to facilitate comparison.
```

Comment:
I think the old sentence is to verbose for a minor point.  

Maybe be compromise:  facilitate comparison of the `b` values?